Formula/mr2.rb

```diff
@@ -0,0 +1,41 @@
+class Mr2 < Formula
+  desc "Expose local server to external network"
+  homepage "https://github.com/txthinking/mr2"
+  url "https://github.com/txthinking/mr2/archive/refs/tags/v20210401.tar.gz"
+  sha256 "3cf2874a5945e79fd9ca270181de1a9d6a662434455c58e2e20e5dbfdebd64c7"
+  license "GPL-3.0-only"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cli/mr2"
+  end
+
+  test do
+    (testpath/"index.html").write <<~EOF
+      <!DOCTYPE HTML>
+      <html>
+      <body>
+        <p>passed</p>
+      </body>
+      </html>
+    EOF
+    mr2_server_port = free_port
+    server_port = free_port
+    client_port = free_port
+    server_pid = fork { exec bin/"mr2", "server", "-l", ":#{mr2_server_port}".to_s, "-p", "password" }
```

Comment:
```suggestion
    server_pid = fork { exec bin/"mr2", "server", "-l", mr2_server_port.to_s, "-p", "password" }
```

Reply:
There is a colon(:) before the port, and it is required. So I can't change it like that.

Reply:
Whoops, apologies for not reading properly. Drop the `.to_s`, then, because the double-quotes around it already make it a string.